app/styles/main.scss

```diff
@@ -1,97 +1,8109 @@
 // NHS.UK frontend
-@import '../../node_modules/nhsuk-frontend/packages/nhsuk';
+@import '../../node_modules/nhsuk-frontend/packages/nhsuk.scss';
+
+
 
 /* ==========================================================================
    #SIDE-NAV
    ========================================================================== */
 
 .app-side-nav__heading {
-  @include nhsuk-font(19, $line-height: 1.25, $weight: bold);
-  color: $nhsuk-secondary-text-color;
+  font-weight: 600;
+  font-size: 16px;
+  font-size: 1rem;
+  line-height: 1.25;
+  color: #4c6272;
   margin-bottom: 12px;
-  padding-top: 4px;
+  padding-top: 4px
+}
+
+@media (min-width: 40.0625em) {
+  .app-side-nav__heading {
+    font-size: 19px;
+    font-size: 1.1875rem;
+    line-height: 1.25
+  }
+}
+
+@media print {
+  .app-side-nav__heading {
+    font-size: 14pt;
+    line-height: 1.25
+  }
+}
+
+@media (max-width: 48.0525em) {
+  .app-side-nav__heading {
+    font-weight: 600;
+    font-size: 20px;
+    font-size: 1.25rem;
+    line-height: 1.25;
+    color: #212b32
+  }
+}
 
-  @include mq($until: desktop) {
-    @include nhsuk-font(24, $line-height: 1.25, $weight: bold);
-    color: $nhsuk-text-color;
+@media (max-width: 48.0525em) and (min-width: 40.0625em) {
+  .app-side-nav__heading {
+    font-size: 24px;
+    font-size: 1.5rem;
+    line-height: 1.25
+  }
+}
+
+@media print and (max-width: 48.0525em) {
+  .app-side-nav__heading {
+    font-size: 18pt;
+    line-height: 1.25
   }
 }
 
 .app-side-nav__back {
-  border-bottom: 1px solid $nhsuk-border-color;
```

Comment:
We should keep these colour variables from the NHS.UK frontend library. If the colour ever changes in the NHS.UK frontend library, the colours will update here automatically without having to manually update all our colours.